docs/source/accelerator/stream.md

```diff
@@ -0,0 +1,88 @@
+# Stream
+
+## Background
+
+Streams are the asynchronous work queues on an accelerator. They allow multiple independent command queues to make progress concurrently on the same device, while the runtime preserves per-thread and per-device “current stream” state. In PyTorch, a stream API must:
+- Provide a per-device default stream and a reusable pool of non-default streams
```

Comment:
Hi, I think not every backend provides a default stream, and the concept of a “default stream” can vary between different backends.
In PyTorch, backends are required to support non-default streams, while the default stream is optional.